docs/logs/customizing-the-sdk/README.md

```diff
@@ -0,0 +1,62 @@
+# Customizing OpenTelemetry .NET SDK for Logs
+
+## OpenTelemetryLoggerProvider
+
+As shown in the [getting-started](../getting-started/README.md) doc,
+calling the extension method `ILoggerBuilder.AddOpenTelemetry` will register
+`OpenTelemetryLoggerProvider` as an `ILoggerProvider` using the default
+configuration. By itself, this is not particularly useful. The subsequent
+sections show how to build a more useful provider.
+
+## OpenTelemetryLoggerProvider configuration
+
+The following concepts can be viewed in the example app
+[Program.cs](Program.cs)
+
+`OpenTelemetryLoggerProvider` holds the logging configuration
+`OpenTelemetryLoggerOptions`, which includes the following:
+
+1. The list of
+   [Processors](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/README.md),
```

Comment:
If we add this, it must link to specification. Should not use collector links, as they are not related.